scipy/sparse/linalg/_isolve/_gcrotmk.py

```diff
@@ -179,7 +179,7 @@ def _fgmres(matvec, v0, m, atol, lpsolve=None, rpsolve=None, cs=(), outer_v=(),
     return Q, R, B, vs, zs, y, res
 
 
-def gcrotmk(A, b, x0=None, tol=1e-5, maxiter=1000, M=None, callback=None,
+def gcrotmk(A, b, x0=None, rtol=1e-5, maxiter=1000, M=None, callback=None,
```

Comment:
Sorry if I missed it in the linked discussion--did the team say they were "ok" with breaking code that currently uses the `tol` keyword argument without a warning/deprecation period? I think this and a few other public functions may be involved in the transition, so just want to make sure this was discussed.

Reply:
> did the team say they were "ok" with breaking code that currently uses the tol keyword argument without a warning/deprecation period?

Thank you @tylerjereddy for your review. It has not been discussed in detailed. Only SYMMLQ function (a new Krylov method) has been discussed in PR #16202. @ilayn